ports/tfc-framework/portfile.cmake

```diff
@@ -14,6 +14,7 @@ endif()
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
+      -DCMAKE_PROJECT_VERSION=${VERSION} # todo I don't like this
```

Comment:
This is crazy, we should maybe move the version back to CMAkelists.txt and I can modify the version sanity checker to parse that file instead.